interview-questions.md

```diff
@@ -4,56 +4,76 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: Object destrucuring is the act of simplifying variable names by setting them to this.state where they are originally initialized.
```

Comment:
Yeah! Its important to note here that object destructuring isn't just for this.state